### PR TITLE
WIP: pom parent and github actions

### PR DESCRIPTION
**Pull request by alexandrem-rte**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Technical evolutions: 
versions and build profiles are inherited from a parent pom (shared with other projects).
github actions are triggered on push

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,6 +4,12 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <parent>
+        <artifactId>farao-parent-ws</artifactId>
+        <groupId>com.farao-community.farao</groupId>
+        <version>1.0.10</version>
+    </parent>
+
     <groupId>org.example</groupId>
```
Comment (TheMaskedTurtle) [inline pom.xml:13]: This should be 'com.farao-community.farao'

File: pom.xml [hunk 2]
```diff
@@ -25,129 +31,10 @@
         <java.version>11</java.version>
```
Comment (TheMaskedTurtle) [inline pom.xml:31]: source encoding and java.version can be removed because they are inherited

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]
I'm not sure you need to import junit-jupiter when you import spring-boot-test

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,13 @@
...
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>org.example</groupId>
+    <parent>
+        <artifactId>farao-parent-ws</artifactId>
+        <groupId>com.farao-community.farao</groupId>
+        <version>1.0.10</version>
+    </parent>
+
+    <groupId>com.farao-community.farao</groupId>
```
Comment (TheMaskedTurtle) [inline pom.xml:13]: Actually you can remove groupId when it is the same as pom parent

Comment (alexandrem-rte) [reply pom.xml]: true!

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**